packages/preferences-persistence/src/migrations/preferences-package-data/convert-editor-settings.js

```diff
@@ -18,6 +18,7 @@ export default function convertEditorSettings( data ) {
 		'showBlockBreadcrumbs',
 		'showIconLabels',
 		'showListViewByDefault',
+		'isPublishSidebarEnabled',
```

Comment:
There's a similar list in the "selectors" of the preferences package that need updating as well.

Reply:
Thanks!